Content.Shared/_Goobstation/CCVars/CCVars.Goob.cs

```diff
@@ -152,4 +149,13 @@ public sealed partial class GoobCVars
 
     public static readonly CVarDef<bool> BlobCanGrowInSpace =
         CVarDef.Create("blob.grow_space", true, CVar.SERVER);
+    
+    #endregion
+
+    #region Mechs
+
+    public static readonly CVarDef<bool> MechGunOutsideMech =
+        CVarDef.Create("mech.gun_outside_mech", true, CVar.REPLICATED);
```

Comment:
```suggestion
        CVarDef.Create("mech.gun_outside_mech", true, Cvar.SERVER | CVar.REPLICATED);
```
Client can shouldn't be able to change this cvar.